deploy/template.yaml

```diff
@@ -263,6 +268,47 @@ Resources:
 
   ### End of API Gateway definition.
 
+  ### API Gateway Custom Domain definition
+
+  F2FApiCustomDomainName:
+    Type: AWS::ApiGateway::DomainName
+    Properties:
+      DomainName: !Sub
+        - "api-${AWS::StackName}.${DNSSUFFIX}"
```

Comment:
Should we include the stack name in all environments?

Reply:
I think this is following DCMAW's pattern - https://github.com/alphagov/di-ipv-dca-back/blob/ae21a7777e2ef34b48da654a9c7f8e1654ced4a5/di-ipv-dca-oauth/template.yaml#L2777